protocol/src/main/protobuf/Transactor.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+package protocol;
+option java_package = "io.mediachain.protocol.transactor";
+
+message MultihashReference {
+    string reference = 1;
+}
+
+message InsertRequest {
+    bytes canonicalCbor = 1;
+}
+
+message UpdateRequest {
+    bytes chainCellCbor = 1;
+}
+
+service TransactorService {
+    rpc FetchObjectChainHead(MultihashReference) returns
```

Comment:
you're right, CanonicalRecord is the right superclass -- I almost wonder if we should print out a big class hierarchy poster